opencast-ilias/OpenCast
# [Release 8] Paella improvements

This PR
fixes #289,
fixes #292,
fixes #293,
fixes #294

### New changes/additions/improvements
- Player configuration now has more unified and cleaner look, by putting options in sections
-  Player configuration now has 4 new options:
   - Prevent video download (#293)
   - Caption Text: Type, Generator, Generator Type (#292)
- Peall player dependency update: (#294)
  - paella-core: 1.48.2
  - paella-basic-plugins: 1.44.7
  - paella-slide-plugins: 1.41.4
  - paella-user-tracking: 1.42.2
  - paella-zoom-plugin: 1.41.3
- No more `restoreLastTime` (#289)

### To test
- Please make sure that your browser `cache` is clear.
- To test captions texts you would need Opencast 15 and auto generated subtitles!


Similar changes will be provided for release_7: https://github.com/opencast-ilias/OpenCast/pull/296
